### PR TITLE
Lethal Injuries - First draft

### PR DESCRIPTION
**Pull request by ulteq**

### CODE REVIEW

File: addons/medical_damage/ACE_Medical_Injuries.hpp [hunk 1]
```diff
@@ -48,11 +51,13 @@ class ACE_Medical_Injuries {
                 minDamage = 0.3;
                 maxDamage = 0.6;
                 bleedingRate = 0.02;
+                lethalitys[] = {{1, 0.10}, {0, 0.20}};
```
Comment (bux578) [inline addons/medical_damage/ACE_Medical_Injuries.hpp]: `lethalities`

**Comment by commy2** [comment]
lgtm

File: addons/medical_damage/functions/fnc_woundsHandler.sqf [hunk 2]
```diff
@@ -39,6 +39,16 @@ call compile _extensionOutput;
 {
     _x params ["", "_woundClassIDToAdd", "_bodyPartNToAdd"];
 
+    private _lethalities = (GVAR(woundsData) select _woundClassIDToAdd) select 7;
```
Comment (Glowbal) [inline addons/medical_damage/functions/fnc_woundsHandler.sqf]: same changes have to be added to `fnc_woundsHandlerSQF`

File: addons/medical_damage/ACE_Medical_Injuries.hpp [hunk 3]
```diff
@@ -48,11 +51,13 @@ class ACE_Medical_Injuries {
                 minDamage = 0.3;
                 maxDamage = 0.6;
                 bleedingRate = 0.02;
+                lethalities[] = {{1, 0.10}, {0, 0.20}};
```
Comment (Glowbal) [inline addons/medical_damage/ACE_Medical_Injuries.hpp]: Not part of this PR, but we will probably want to add macros for the body part indices to get rid of the magic numbers.

File: addons/medical_damage/ACE_Medical_Injuries.hpp [hunk 4]
```diff
@@ -169,20 +178,23 @@ class ACE_Medical_Injuries {
...
-                maxDamage = 0.75;
+                minDamage = 3.5;
+                maxDamage = 8;
                 bleedingRate = 0.05;
+                lethalities[] = {{1, 0.15}, {0, 0.60}};
             };
             class Large {
                 name = CSTRING(VelocityWound_Large);
-                minDamage = 0.75;
+                minDamage = 8;
```
Comment (Glowbal) [inline addons/medical_damage/ACE_Medical_Injuries.hpp]: this seems incredibly high - did you test this value and if so, what were the results?

**Comment by ulteq** [comment]
I will squash everything and make a new pull request.

**Comment by jonpas** [comment]
-> #4744
